# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/jfsutils/hotfix-paths.patch.musl
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- jfsutils-1.1.15/libfs/fssubs.c.vanilla	2022-09-06 18:02:33.465163406 +0200
+++ jfsutils-1.1.15/libfs/fssubs.c	2022-09-06 18:02:42.955163559 +0200
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <paths.h>
 
 #if HAVE_MNTENT_H
 #include <mntent.h>
@@ -31,7 +32,6 @@
 #endif
 
 #if HAVE_GETMNTINFO
-#include <paths.h>
 #include <sys/param.h>
 #include <sys/mount.h>
 #endif
